scripts/check_e2e_tests_are_captured_in_ci.py

```diff
@@ -28,7 +28,7 @@
 # These test suites are not present in CI. One is extra
 # (ie. (full: [*.js])) and other test suites are being run by CircleCI.
 TEST_SUITES_NOT_RUN_IN_CI = [
-    'full', 'adminPage', 'library', 'publication',
+    'full', 'library',
```

Comment:
[Out of context of this PR]
@U8NWXD We should audit one to ensure we are running all the e2e tests and have a strict check to ensure all the e2e tests are running on CII.

Reply:
I filed #12638 to track this